### ISSUE TITLE
Make simple search cap-insensitive by default

### ISSUE THREAD

**Issue by andrew-morris** [nicksherron/bashhub-server#9]
Right now the search functionality is cap-sensitive. This makes sense when a user is providing a regex, but for simple/quick search, it makes more sense to make search terms cap-insensitive

**Comment by nicksherron** [comment]
Good idea, keep em coming! Just pushed this in  eeb43dfbbe93642f05fad996eb3a3658f6261d00.

**Comment by nicksherron** [comment]
I had to revert the changes because regex wouldn't work with the way it was implemented. Still open to doing this but not sure it's possible without effecting regex.
